src/core/zowe/core_for_zowe_sdk/session.py

```diff
@@ -0,0 +1,71 @@
+"""Zowe Python Client SDK.
+
+This program and the accompanying materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this distribution, and is available at
+
+https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zowe Project.
+"""
+
+from dataclasses import dataclass
+from typing import Union
+
+from zowe.core_for_zowe_sdk.zosmf_profile2 import ProfileManager
+from . import session_constants
+
+
+@dataclass
+class ISession:
+    """
+    Class to represent session parameters
+    """
+
+    hostname: str
+    port: int
+    rejectUnauthorised: bool = True
```

Comment:
Please change the spelling to "rejectUnauthorized" in all places where this is used.

I know this is a difference between British and American spelling, but we should be consistent with existing Zowe code that expects it to be spelled a particular way 🙂